scripts/load_cores.sh

```diff
@@ -162,7 +162,10 @@ do
     ${NAMING_CONSORTIUM_PARAM} ${ADD_CONSORTIUM_NAME_PARAM}
 
   perl -Ilocal/lib/perl5 ${ENSDIR}/ensembl-hive/scripts/beekeeper.pl -url mysql://${HIVE_USER}:${HIVE_PASS}@${HIVE_HOST}:${HIVE_PORT}/${HIVE_DB} -loop
-
+  
+  echo "Updating 5' and 3' UTR checksum in translation table after loading ${CORE_DB} ..."
```

Comment:
@ranjit-s Why update after loading? Why not do it as part of loading?

Reply:
Hi @sanjay-boddu , the update is done after loading, so that UTR checksum in translation table of all rows are checked (and updated if required) and not just those which were impacted by this release.